web/tailwind.config.js

```diff
@@ -2,7 +2,17 @@ module.exports = {
     purge: ['./src/**/*.js', './src/**/*.jsx'],
     darkMode: false, // or 'media' or 'class'
     theme: {
-        extend: {},
+        extend: {
+            colors: {
+                'brand-darkest-blue': '#12163B',
+                'brand-dark-blue': '#282768',
+                'brand-red': '#E81E25',
+                'brand-light-blue': '#6BABE5',
+                'brand-lightest-blue': '#ACC9E8',
+                'brand-purple': '#3F2B56',
+                'brand-light-purple': '#D4C0CD',
```

Comment:
Let's keep to the number system `brand-1`. In the future if they change colors we won't have to worry about changing the variable name too.

Reply:
But then, in the future, when `brand-darkest-blue` gets changed to orange, we'll either have to live with the variable name `-blue` being associated with an orange color or do a "find and replace" on the whole project to update the variable name to something important. Neither option sounds good to me.

Reply:
@asasmith Find and replace would still be more efficient than having to check every time I need to use a color 😅